src/overlays/actors/ovl_En_Holl/z_en_holl.h

```diff
@@ -4,6 +4,33 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define ENHOLL_GET_TYPE(thisx) (((thisx)->params >> 6) & 7)
+#define ENHOLL_GET_SWITCH_FLAG(thisx) ((thisx)->params & 0x3F)
+
+/**
+ * Two kinds of holls:
+ *   Horizontal (H):
+ *     For the player to move horizontally between rooms.
+ *     Cuboid-shaped, 200 units high plus 50 units into the ground, 400 units wide or, if narrow, 200 units wide.
```

Comment:
theres alot going on in this sentence. maybe
```
Cuboid-shaped: 
- 200 units high, plus 50 unit into the ground
- 400 units wide, or 200 units if narrow
```
or something similar to break the information up